components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -317,6 +319,20 @@ export function CreateWorkspacePage() {
         );
     }
 
+    if (needsGitAuthorization) {
+        return (
+            <div className="flex flex-col mt-32 mx-auto ">
+                <div className="flex flex-col max-h-screen max-w-lg mx-auto items-center w-full">
+                    <Heading1>New Workspace</Heading1>
+                    <div className="text-gray-500 text-center text-base">
+                        Start a new workspace with the following options.
+                    </div>
+                    <AuthorizeGit />
```

Comment:
**thought**: If we use this point of control for adding the integration, we no longer need to hide the workspaces and projects menu items in the top navigation. ✨